geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/package.scala

```diff
@@ -28,6 +28,8 @@ package object geotools {
   val CRS_EPSG_4326: CoordinateReferenceSystem =
     try { CRS.decode("EPSG:4326", true) } catch { case t: Throwable => DefaultGeographicCRS.WGS84 }
 
+  val CrsEpsg4326: CoordinateReferenceSystem = CRS_EPSG_4326
```

Comment:
Anything wrong with the old val?

Reply:
nah, just trying to standardize on CamelCase constants.